web/api.go

```diff
@@ -323,3 +324,32 @@ func ApiSAPSystemDeleteTagHandler(client consul.Client) gin.HandlerFunc {
 		c.JSON(http.StatusNoContent, nil)
 	}
 }
+
+// ApiCheckResultsHandler godoc
+// @Summary Get a specific cluster's check results
+// @Produce json
+// @Success 200 {object} map[string]interface{}
+// @Error 500
+// @Router /api/clusters/{id}/results
+func ApiCheckResultsHandler(client consul.Client, s services.ChecksService) gin.HandlerFunc {
+	return func(c *gin.Context) {
+		clusterId := c.Param("id")
```

Comment:
This `id` might be a bit confusing/ What about `cluster_id`?

Or otherwise we could rename the func name to be specific for the cluster. I forsee other checks out of the cluster topic in the future

Reply:
Addressed, you're right